src/core/sys/darwin/sys/event.d

```diff
@@ -43,7 +43,6 @@ enum : short
     EVFILT_FS       =  -9, /* filesystem events */
     EVFILT_USER     = -10, /* User events */
     EVFILT_VM       = -12, /* virtual memory events */
-    EVFILT_SYSCOUNT =  11
```

Comment:
OK, if you can't use it in API calls, I think this is fine to remove without deprecation cycle.